src/util/projects.tsx

```diff
@@ -13,7 +13,7 @@ export const projectSubpages = [
 export const getSubpageFromUrl = (url: string) => {
   const parts = url.split("/");
   if (projectSubpages.includes(parts[4])) {
-    return parts[4];
+    return parts.slice(4).join("/");
```

Comment:
This is too generic, i.e. when on a detail page instance, profile, network, etc. This asumes the detail page will exist for any project, which is not true, only for some subpages under the storage tab.

Reply:
Good point, I made it more specific. It should work as expected now.